### ISSUE TITLE
asking for sudo password upon login

### ISSUE THREAD

**Issue by kronikpillow** [zimfw/zimfw#427]
```
zimfw version: 1.4.3 (built at 2021-03-19 23:42:38 UTC, previous commit is 163d36b)
ZIM_HOME:      /home/kronikpillow/.config/zsh/.zim
Zsh version:   5.8
System info:   Linux susanoo 5.11.9-2-ck-ivybridge #1 SMP PREEMPT Wed, 24 Mar 2021 21:34:40 +0000 x86_64 GNU/Linux

```
my .zimrc .zshenv .zprofile and .zshrc
`https://github.com/kronikpillow/voidrice/tree/master/.config/zsh`

Description
-----------

I used to use luke smiths larbs, and have some of his dotfile settings as key of my own dotfiles now, recently i decided to see how far zimfw has gone with changes and to test out the latest version, after installing it, i am getting prompted for my sudo password right after login, even when connecting from a different machine ... this does not happen when i uninstall zim and switch back to my standard .zprofile and .zshenv which is not much different from what it is now .. (only difference being the .zshrc it self and having ZIM HOME variable in zshenv

`ssh kronikpillow@susanoo -p 2222
kronikpillow@susanoo's password: 
Last login: Sat Mar 27 08:39:31 2021 from 93.87.123.179
[sudo] password for kronikpillow: 
`

checkout my .zshenv .zprofile .zinitrc and .zshrc please and tell me what could be causing this?

**Comment by ericbn** [comment]
I see this on your zprofile:
```
sudo -n loadkeys ${XDG_DATA_HOME:-$HOME/.local/share}/larbs/ttymaps.kmap 2>/dev/null
```
And there's much more going on in your startup files than just setting up Zim, so it could be any other thing...

**Comment by kronikpillow** [comment]
yeah i figure it's the loadkeys function, but loadkeys is set in /etc/sudoers to run without password, while at the same time that command didn't create any issues until i switched to zim itself, so im wondering how to debug what is causing the sudo password prompt for sure? I mean why is zimfw triggering it, while zsh without it isn't ... it was a identical setup, i even merged my original .zshrc which was actually [luke smith's zshrc](https://github.com/LukeSmithxyz/voidrice/blob/master/.config/zsh/.zshrc)

**Comment by kronikpillow** [comment]
investigating further, I have made a commit disabling loadkeys, still a sudo login prompt upon login :-/

**Comment by kronikpillow** [comment]
ok ... narrowed it down to `[ ! -f ${XDG_CONFIG_HOME:-$HOME/.config}/shell/shortcutrc ] && shortcuts >/dev/null 2>&1 &` ... which i find strange ...  here is[shortcuts script](https://github.com/kronikpillow/voidrice/blob/master/.local/bin/shortcuts) ... can someone help me figure this out please? I know it's probably a pita and a newbie situation, but this same setup works without zimfw so I naturally came for help here first :) or point me in the right direction? thanks 😄😄

**Comment by kronikpillow** [comment]
ok i was wrong ... it wasn't that command ... i have went one by one in my .zshenv and .zprofile and enabled disabled stuff ... then i'v gone to my xinitrc xprofile and enabled disabled stuff there ... what changed was that on my display sometimes it asked for the sudo password some times it didn't (im assuming it cached the previous sudo password input and didn't ask for it every time) ... but every time i ssh in to my machine, it keeps asking for my sudo password on login ... then i uninstalled zimfw, and reverted to my default .zshrc and problem is completely gone, on login and on ssh login ... 

now i'm going to reinstall zimfw and see what happens
